code/script.js

```diff
@@ -27,7 +34,7 @@ const CHARACTERS = [
     hair: 'grey',
     eyes: 'blue',
     accessories: ['hat'],
-    other: ['smoker']
+    other: ['smoker', 'agile-coach']
```

Comment:
Nice that you added more features to the characters. Now I am curious why Jacques is an agile coach :)